spec/rubocop/cop/lint/return_in_void_context_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::ReturnInVoidContext do
+  subject(:cop) { described_class.new }
+
+  shared_examples 'registers an offense' do |message|
+    it 'registers an offense' do
+      inspect_source(cop, source)
+
+      expect(cop.offenses.size).to eq(1)
+      expect(cop.messages).to eq([message])
+    end
+  end
+
+  shared_examples 'does not registers an offense' do
+    it 'does not registers an offense' do
+      inspect_source(cop, source)
+      expect(cop.offenses.size).to eq(0)
+    end
+  end
+
+  context 'with an initialize method containing a return with a value' do
+    let(:source) do
+      ['class A',
```

Comment:
Change all the code examples to heredocs. We've recently changed them in all specs.